hack/pre-release.sh

```diff
@@ -329,6 +329,8 @@ gitlab_update() {
   wmco_version=$2
   updated_version=$3
   OCP_version=$(get_OCP_version $wmco_version)
+  updated_OCP_version=$(get_OCP_version $updated_version)
```

Comment:
Something is not clear for me;  does `updated` in ` updated_OCP_version` means the version was updated already?   Reading the code; looks like this is referring to a new (target) OCP version that will replace the existing one (`OCP_version`)

Reply:
yeah, that's the target OCP version that will replace the existing one. Maybe I should change "updated" to "target" throughout the file, because that does sound a lot better...  :thinking:

Reply:
though it may be better to do that in a different follow-up.